MVCapplication/src/main/java/com/softserve/if072/mvcapp/controller/ProductPageController.java

```diff
@@ -29,15 +28,24 @@
 @PropertySource(value = {"classpath:application.properties"})
 public class ProductPageController {
 
-    @Value("${application.restProductURI}")
-    private String productUri;
+    @Value("${application.restProductURL}")
+    private String productUrl;
+
+    @Value("${application.restUnitURL}")
+    private String unitUrl;
+
+    @Value("${application.restCategoryURL}")
+    private String categoryUrl;
+
+    private Unit[] unitResult;
+    //private Category[] categoryResult;
 
     @RequestMapping("/")
-    public String getProductPage(ModelMap model){
+    public String getProductPage(ModelMap model) {
 
         int userId = 1;
 
-        final String uri = new String(productUri + "/user/{userId}");
+        final String uri = new String(productUrl + "/user/{userId}");
```

Comment:
Guys, This is common issue. DO NOT wrap `productUrl + "/user/{userId}"` into new String object